aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/article_detail.html

```diff
@@ -4,6 +4,13 @@
 {% block title %}{{ article.title }} - {{ block.super }}{% endblock %}
 {% block breadcrumb %}{% endblock %}
 
+{% block html_tag %} itemscope itemtype="http://schema.org/Blog" {% endblock %}
+
+{% block meta_tags %}
+    <meta name="viewport" content="width=device-width,initial-scale=1">
```

Comment:
To elaborate further: https://github.com/aldryn/aldryn-newsblog/blob/master/aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/base.html#L3 is the standard template. If you want meta integration you just adapt the base.html template to look like:

```
{% block meta_tags %}
    {{ block.super }}
    {% block newsblog_meta_tags %}{% endblock %}
{% endblock %}
{% block content %}{% endblock content %}
```

and the article_detail.html just provides the facilities to overwrite it:

```
{% block newsblog_meta_tags %}
    {% if meta %}
        {% include "aldryn_newsblog/includes/meta.html" with meta=meta %}
    {% endif %}
{% endif %}
```